sk0-back/src/users/dtos/create-user.dto.ts

```diff
@@ -24,4 +26,8 @@ export class CreateUserDto {
   @IsString()
   @MinLength(3)
   name: string;
+
+  @IsOptional()
+  @IsNumber()
+  role?: number = 0;
```

Comment:
I think we should not allow people to set their roles. We should have dedicated admins later, who will handle those.

Reply:
Roles should be strings or mapped from numbers to strings. It is easier to run into a falsy value, than accidentally create an "ADMIN", then a `0`.

Reply:
Got it. I thought setting it to 0 (user) by default was okay, since the user cant set it's role during registration. This way the only way to crea a role 1 (asmin) user was manually. Though no one stops someone to add the role:1 in postman... Ill delete the role parts and references.